Articles/Blog/RseStoriesPodcast.md

```diff
@@ -10,75 +10,73 @@
 
 ### We look to experts for knowledge, but do they know the whole picture?
 
-I woke up one morning in September 2019 with something on my mind. I have read several
-papers and heard talks and other "academically approved" presentation formats that purport to define what
-a research software engineer (RSE) is. But the definitions don't reflect my experience.
```

Comment:
Comments for reverting back:
 - To start in a past tense (I woke up one morning) and jump into present (I have read...) makes no sense.
 - I would never use the word "purport" and I don't think this sounds natural in spoken or written speech. 
 - I would never start a sentence with "Nor" and I don't think that is a full sentence either.
 - I purposefully asked questions to the reader (and to myself at the time) as a strategy / style, and do not wish for this to be removed in favor of "formality."